File: youlexuan-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function ($http) {

    this.search = function (page,rows,searchEntity) {
        return $http.post('../brand/search.do?page='+page+'&rows='+rows,searchEntity);
    }


    this.findPage = function (page,rows) {
        return  $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
    }

    this.dele = function (selectIds) {
        return $http.get('../brand/dele.do?selectIds='+selectIds);
    }

    this.save = function (methodName,entity) {
        return $http.post('../brand/'+ methodName +'.do',entity)
    }

    this.findOne  = function (id) {
        return $http.get('../brand/findOne.do?id='+id)
    }

    this.findAll = function () {
        return $http.get('../brand/findAll.do')
    }

    this.selectOptionList = function () {

        return $http.get('../brand/selectOptionList.do')
    }

})

File: youlexuan-sellgoods-interface/src/main/java/com/offcn/sellergoods/service/BrandService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {

           public List<TbBrand> findAll();


    PageResult findPage(int page, int rows);

    void add(TbBrand tbBrand);

    void update(TbBrand tbBrand);

    TbBrand findOne(Long id);

    void dele(Long[] arr);

    PageResult search(TbBrand tbBrand, Integer page, Integer rows);

    List<Map> selectOptionList();
}
